src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -356,4 +375,11 @@ public Boolean isStarted() {
         }
         return startDate.isBefore(ZonedDateTime.now());
     }
+
+    public Boolean isPublished() {
```

Comment:
*isPublished* might be a little bit confusing with regard to *isVisible*. I guess something like *resultsPublished* would be more intuitive here.

Reply:
Done